code/modules/reagents/newchem/drugs.dm

```diff
@@ -551,7 +551,7 @@
 /datum/reagent/lube/ultra/addiction_act_stage4(var/mob/living/carbon/human/M as mob)
 	M.Jitter(20)
 	M.Dizzy(20)
-	M.adjustToxLoss(5)
+	M.adjustBrainLoss(2)
```

Comment:
Just checked; they have the same problem brains did; their damage caps at 60.

You'll have to set the MMI holder's max health to 200, same as brains, so it has the same impacts/effects.

Also, you'll probably want to bump up the brain damage here by 25%---2 brainloss will not be 2 brainloss; it'll be 1.8 because all robotic organs have 20% damage resist; 2.5 brainloss will effectively deal 2 brain damage per cycle.